### PR TITLE
Feature/add maturin

### PR DESCRIPTION
**Pull request by simobasso**
Hey @messense! I've just rebased your pr if you need it again!

@xrmx, do you have time for a review 😏  ?

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,29 +2,7 @@ FROM python:3.6-alpine as base-image
...
-    rm "/etc/apk/keys/sgerrand.rsa.pub" && \
-    /usr/glibc-compat/bin/localedef --force --inputfile POSIX --charmap UTF-8 "$LANG" || true && \
-    echo "export LANG=$LANG" > /etc/profile.d/locale.sh && \
-    apk del glibc-i18n && \
-    rm \
-        "/root/.wget-hsts" \
-        "$ALPINE_GLIBC_BASE_PACKAGE_FILENAME" \
-        "$ALPINE_GLIBC_BIN_PACKAGE_FILENAME" \
-        "$ALPINE_GLIBC_I18N_PACKAGE_FILENAME"
+RUN apk add libgcc
```
[the remark this answers is not in the document]
Comment (simobasso) [reply Dockerfile:5]: Imho we can safely remove it in another pr

File: tox.ini [hunk 2]
```diff
@@ -8,11 +8,10 @@ deps =
     py36: black==19.3b0
     -rrequirements/requirements_test.txt
     -rrequirements/requirements.txt
-    pyo3-pack==0.6.1
-    setuptools-rust==0.11.6
+    maturin==0.8.2
```
Comment (xrmx) [inline tox.ini:11]: We should probably match the dockerfile with 0.8.3 here. Note to self, more recent versions drop Python 3.5 support, which we can do later i guess.

File: .travis.yml [hunk 3]
```diff
@@ -14,7 +15,7 @@ install:
   - chmod +x rustup-init
   - ./rustup-init -y --no-modify-path --default-toolchain stable
   - chmod -R a+w $RUSTUP_HOME $CARGO_HOME
-  - pip install tox-travis tox pyo3-pack==0.6.1 setuptools-rust==0.11.6
+  - pip install tox-travis tox maturin
```
Comment (xrmx) [inline .travis.yml:18]: do we really need to install maturin here? Isn't already installed in tox?

**Review by xrmx — APPROVED** [review]
I've tested I can successfully build sdist and wheels for python 3.8 and 3.9 with maturin 0.8.3 on this branch, so other than the two small comments LGTM.

File: tox.ini (hunk 2 above)
Comment (simobasso) [reply tox.ini:11]: thanks!

File: .travis.yml (hunk 3 above)
Comment (simobasso) [reply .travis.yml:18]: let's try!

File: tox.ini (hunk 2 above)
Comment (simobasso) [reply tox.ini]: done in https://github.com/OvalMoney/celery-exporter/pull/43/commits/a600564a985cfff98cd00bb5d85f6d629979bcdf

**Comment by SharpEdgeMarshall** [comment]
lgtm, can we merge?

File: Dockerfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (simobasso) [reply Dockerfile:5]: let's continue on https://github.com/OvalMoney/celery-exporter/issues/45

**Comment by simobasso** [comment]
@xrmx 3.9 is not tested https://github.com/OvalMoney/celery-exporter/issues/46

**Comment by xrmx** [comment]
@simobasso sure, maturin builds a wheel for whatever python version it finds :)

**Pull request merged**